src/order.rs

```diff
@@ -491,3 +485,71 @@ impl fmt::Debug for KeyAuthorization {
         f.debug_tuple("KeyAuthorization").finish()
     }
 }
+
+/// A policy for retrying API requests
+///
+/// Refresh the order state from the server for `tries` times, waiting `delay` before the
+/// first attempt and increasing the delay by a factor of 2 for each subsequent attempt.
+///
+/// The `Default` implementation will retry 5 times with an initial delay of 250ms.
+#[derive(Debug, Clone, Copy)]
+pub struct RetryPolicy {
+    tries: u8,
+    delay: Duration,
+}
+
+impl RetryPolicy {
+    /// A `const` constructor for the default `RetryPolicy`
```

Comment:
very nit-picky nit: I don't think it adds much to mention the `const` aspect since its self-evident from the fn signature.

Reply:
Fair.